CMakeLists.txt

```diff
@@ -556,6 +572,41 @@ if(QUDA_DOWNLOAD_USQCD)
   find_program(AUTORECONF_EXE NAMES autoreconf)
 endif()
 
+if(QUDA_NVSHMEM)
+  if(QUDA_DOWNLOAD_NVSHMEM)
+    ExternalProject_Add(NVSHMEM
+                        URL ${QUDA_DOWNLOAD_NVSHMEM_TAR}
+                        PREFIX nvshmem
+                        CONFIGURE_COMMAND ""
+                        BUILD_IN_SOURCE ON
+                        BUILD_COMMAND  make NVSHMEM_PREFIX=<INSTALL_DIR> NVSHMEM_MPI_SUPPORT=1 GDRCOPY_HOME=/usr/local/gdrcopy/ install
```

Comment:
Add a cmake flag for overriding the default `gdrcopy` path

Reply:
done.